model_server/run_local.sh

```diff
@@ -18,7 +18,7 @@ python3 -m venv .venv
 source .venv/bin/activate
 cd model_server
 python3 -m pip install --upgrade pip
-pip3 install torch==2.2.2 --extra-index-url https://download.pytorch.org/whl/cpu
+pip3 install torch==$(grep '^torch==' ../nl_requirements.txt | cut -d'=' -f3) --extra-index-url https://download.pytorch.org/whl/cpu
```

Comment:
can we add the requirement to model_server/requirements.txt. I don't think this one needs to be in sync with the others since we don't really use this.

also can you look into whether this line needs to be updated: https://github.com/datacommonsorg/website/blob/e71bdece22de5c86f1dd646b2e7e73d865ca10a1/model_server/Dockerfile#L16C45-L16C56
which version of pytorch does that pull in?